cloud-operator/templates/role.yaml

```
@@ -21,4 +21,15 @@ rules:
 - apiGroups: [""]
   resources: ["secrets"]
   resourceNames: ["hubble-relay-client-certs"] # Name of the Secret
-  verbs: ["get"]
\ No newline at end of file
+  verbs: ["get"]
+---
+# Role to read secrets in the GKE managed observability namespace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: illumio-cloud-operator-secret-reader
```

Use the same name as the Role we already create in namespace `kube-system`, for consistency.
```suggestion
  name: hubble-client-certs-reader
```